src/main/java/com/danielagapov/spawn/Controllers/EventController.java

```diff
@@ -25,6 +27,12 @@ public Event getEvent(@PathVariable Long id) {
         return eventService.getEventById(id);
     }
 
+    // full path: /api/v1/events/user/{id}
+    @GetMapping("events/{userID}")
```

Comment:
This mapping should match your full path; I think you might've made a typo

```suggestion
    @GetMapping("events/user/{userID}")
```